src/test/java/com/edu/ComputerNumberTest.java

```diff
@@ -0,0 +1,35 @@
+package com.edu;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class ComputerNumberTest {
+
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"123", "456", "789", "678", "146", "964"})
+    void comNumberTrueTest(String testNumber) {
+        assertThatCode(() -> {
+            user.validUserNumber(testNumber);
+        }).doesNotThrowAnyException();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1213", "56", "9", "abc", "a12", "ㅂㅁㅋ"})
+    void comNumberFalseTest(String testNumber) {
```

Comment:
```suggestion
    void comNumberFalseTest(String userNumber) {
```

테스트 코드의 파라미터명 혹은 변수명도 네이밍에 신경 써주셔야 합니다. 네이밍이 힘든 경우라면 testNumber와 같은 네이밍도 조금은 묵인할 수 있겠지만 지금같은 테스트는 파라미터의 네이밍을 명백하게 정할 수 있을 것 같습니다.